Easydict/NewApp/View/MenuView/MainMenuCommand.swift

```diff
@@ -9,8 +9,21 @@
 import SwiftUI
 
 struct EasyDictMainMenu: Commands {
+    @Environment(\.openURL)
+    private var openURL
+
     var body: some Commands {
         // shortcut
         MainMenuShortcutCommand()
+
+        // Help
+        CommandGroup(replacing: CommandGroupPlacement.help, addition: {
+            Button("menu_feedback") {
+                guard let versionURL = URL(string: "\(EZGithubRepoEasydictURL)/issues") else {
+                    return
+                }
+                openURL(versionURL)
```

Comment:
```suggestion
                openURL(URL(string: "\(EZGithubRepoEasydictURL)/issues")!)
```

Personally, I think it's safe to force unwrap a hardcode url here.

Reply:
But if the URL error, the App will open a blank page in web.

Reply:
This hardcoded string should not fail to init in this case. Otherwise it will generate a fatal error.

Reply:
yeah, I think you are right!